feat:  Groups Management documentation (#125)
mia-platform/documentation: docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md

title:  Console levels and permission management
sidebar_label: Console Levels and Permission Management
---
Assigning different Roles and permissions to each [Identity](/development_suite/identity-and-access-management/overview.md#identity-and-access-management) that has access to Mia-Platform Console is a key action for defining responsibilities within your platform ecosystem.

> The Console is based on hierarchical levels, and for each of them, specific permissions and capabilities have been identified and can be assigned.  

Reviewer: ```suggestion
The Console is based on hierarchical levels and, for each of them, specific permissions and capabilities have been identified and can be assigned.  
```

## Identity Capabilities inside Console

> Identities can perform a set of predetermined actions along Console levels, in accordance with a set of permissions grouped within a specific role. A set of default Roles has been defined:

Reviewer: ```suggestion
Identities can perform a set of predetermined actions along Console levels, in accordance with a set of permissions grouped for each specific role. A brief presentation of the default Roles that can be assigned to each Identity is provided here below:
```

> Identities can perform a set of predetermined actions along Console levels, in accordance with a set of permissions grouped for each specific role. A brief presentation of the default Roles that can be assigned to each Identity is provided here below:

Author: ```suggestion
Identities can perform a set of predetermined actions along Console levels, in accordance with a set of permissions grouped for each specific role. A brief presentation of the default roles that can be assigned to each Identity is provided here below:
```

Author: Why does the Role term start with uppercase?
